Sources/WasmInterpreter/WasmInterpreter+ImportHandler.swift

```diff
@@ -693,4 +693,134 @@ extension WasmInterpreter {
         )
         try self.importNativeFunction(named: name, namespace: namespace, signature: sig, handler: importedFunction)
     }
+    
+    public func addImportHandler<Arg1, Arg2, Arg3, Arg4, Arg5, Arg6, Arg7, Ret>(
+        named name: String,
+        namespace: String,
+        block: @escaping (Arg1, Arg2, Arg3, Arg4, Arg5, Arg6, Arg7) throws -> Ret
+    ) throws
+        where Arg1: WasmTypeProtocol, Arg2: WasmTypeProtocol, Arg3: WasmTypeProtocol,
+              Arg4: WasmTypeProtocol, Arg5: WasmTypeProtocol, Arg6: WasmTypeProtocol, Arg7: WasmTypeProtocol, Ret: WasmTypeProtocol
+    {
+        let importedFunction: ImportedFunctionSignature =
+        { (stack: UnsafeMutablePointer<UInt64>?, heap: UnsafeMutableRawPointer?) -> UnsafeRawPointer? in
+            do {
+                let arg1: Arg1 = try NativeFunction.argument(from: stack, at: 0)
```

Comment:
With v0.5.0, [Wasm3 now puts the function arguments after the return value on the stack](https://github.com/wasm3/wasm3/blob/067261d490a1cbcfba1471c3c31af61197aacdd8/source/wasm3.h#L241). So, we need to increment each of these indexes by one.

Reply:
@atdrendel would you like me to increment each of the indexes for all functions that have a return value, or just the ones that I've added in this PR? Not sure if this is something best done now, or as a part of a wasm3 v0.5.0 upgrade PR

Reply:
If you pull main into this branch, then you'll only need to update the ones you've added. I've already merged the 0.5.0 changes into main.

Reply:
Perfect. I went ahead and rebased on main and incremented the indexes for the functions with a return value